### ISSUE TITLE
Unable to list scripts if `post-install-cmd` or `post-update-cmd` defined

### ISSUE THREAD

**Issue by jvis** [composer/composer#7188]
My `composer.json`:

```json
{
    "name": "jvis/test",
    "description": "test",
    "authors": [
        {
            "name": "Josh",
            "email": "contact@jvinternetsolutions.com"
        }
    ],
    "require": {},
    "require-dev": {},
    "scripts": {
        "post-install-cmd": "@php -v",
        "post-update-cmd": "@php -v"
    }
}
```

Output of `composer diagnose`:

```
Checking composer.json: WARNING
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.
Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com oauth access: OK
Checking disk free space: OK
Checking pubkeys:
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: OK
Composer version: 1.6.3
PHP version: 7.1.11
PHP binary path: C:\dev\php\7.1\php.exe
```

When I run this command:

```
composer run-script --list
```

I get the following output:

```
scripts:


  [Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "post-install-cmd" is not defined.


run-script [--timeout TIMEOUT] [--dev] [--no-dev] [-l|--list] [--] [<script>] [<args>]...
```

And I expected this to happen:

List the defined scripts

**Comment by jvis** [comment]
```json
{
    "name": "jvis/test",
    "description": "test",
    "authors": [
        {
            "name": "Josh",
            "email": "contact@jvinternetsolutions.com"
        }
    ],
    "require": {},
    "require-dev": {},
    "scripts": {
        "post-update-cmd": "@php -v"
    }
}
```

```bash
$ composer run-script --list
scripts:


  [Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "post-update-cmd" is not defined.


run-script [--timeout TIMEOUT] [--dev] [--no-dev] [-l|--list] [--] [<script>] [<args>]...
```

**Comment by jvis** [comment]
```json
{
    "name": "jvis/test",
    "description": "test",
    "authors": [
        {
            "name": "Josh",
            "email": "contact@jvinternetsolutions.com"
        }
    ],
    "require": {},
    "require-dev": {},
    "scripts": {
        "check": "@php -vs"
    }
}
```
```bash
$ composer run-script --list
scripts:
  check  Runs the check script as defined in composer.json.
```

**Comment by Seldaek** [comment]
Fixed in ec9ba46c5f6ba537bc11a2f33d6c4483252c564c
